docs/static/releasenotes.asciidoc

```diff
@@ -61,23 +61,209 @@ This section summarizes the changes in the following releases:
 
 [[logstash-7-17-0]]
 === Logstash 7.17.0 Release Notes
-coming[7.17.0]
 
 [[featured-7-17-0]]
 ==== New features and enhancements
 
-
+* Docker images for `full` and `oss` distributions are now is based on Ununtu 20.04 (`ubi8` image remains unchanged) https://github.com/elastic/logstash/pull/13529[#13529]
+//* Backport #13442 to 7.17. Update logstash docker to use ubuntu 20.04 b… https://github.com/elastic/logstash/pull/13529[#13529]
 
 [[notable-7-17-0]]
 ==== Performance improvements and notable issues fixed
 
+* Reduces the scope of a memory leak that can be caused by processing events with UUID or other high-cardinality field names. https://github.com/elastic/logstash/pull/13655[#13655]
+* Fix an error in the `logstash-plugin install` command that prevented the installation of non-default plugins https://github.com/elastic/logstash/pull/13405[#13405]
+* Fixed an issue where invoking the Logstash Keystore Utility (`bin/logstash-keystore`) incorrectly set up a logging directory to the literal `${sys:ls.logs}` and produced noise to the console about routing logs. This utility now correctly configures its logger using the provided settings file. https://github.com/elastic/logstash/pull/13221[#13221]
+* Fixes `events.out` metric count when there the events are dropped in filter-output section of the pipeline. Events that were dropped are no longer included. https://github.com/elastic/logstash/pull/13593[#13593]
+* Fixes a regression introduced in `7.12`, in which the `LS_JAVA_OPTS` environment variable is ignored when a readable `jvm.options` file cannot be found. https://github.com/elastic/logstash/pull/13525[#13525]
+* Fix a crash of Logstash in initialization when using the `logstash-input-azure_eventhub` plugin. https://github.com/elastic/logstash/pull/13603[#13603]
+* Fixes an issue where environment variables in pipeline condition statements were not substituted https://github.com/elastic/logstash/pull/13608[#13608]
+* Reduces Deprecation noise in the Elastic Upgrade Assistant. Elasticsearch requests made by the Monitoring or Central Management features that are not directly user-configurable will include a special product origin header so that Upgrade Assistant can avoid calling out deprecations that are not user-actionable. https://github.com/elastic/logstash/pull/13563[#13563]
```

Comment:
```suggestion
* Reduced the scope of a memory leak that can be caused by processing events with UUID or other high-cardinality field names. https://github.com/elastic/logstash/pull/13655[#13655]
* Fixed an error in the `logstash-plugin install` command that prevented the installation of non-default plugins https://github.com/elastic/logstash/pull/13405[#13405]
* Fixed an issue where invoking the Logstash Keystore Utility (`bin/logstash-keystore`) incorrectly set up a logging directory to the literal `${sys:ls.logs}` and produced noise to the console about routing logs. This utility now correctly configures its logger using the provided settings file. https://github.com/elastic/logstash/pull/13221[#13221]
* Fixed `events.out` metric count when there the events are dropped in filter-output section of the pipeline. Events that were dropped are no longer included. https://github.com/elastic/logstash/pull/13593[#13593]
* Fixed a regression introduced in `7.12`, in which the `LS_JAVA_OPTS` environment variable is ignored when a readable `jvm.options` file cannot be found. https://github.com/elastic/logstash/pull/13525[#13525]
* Fixed a crash of Logstash in initialization when using the `logstash-input-azure_eventhub` plugin. https://github.com/elastic/logstash/pull/13603[#13603]
* Fixed an issue where environment variables in pipeline condition statements were not substituted https://github.com/elastic/logstash/pull/13608[#13608]
* Reduced Deprecation noise in the Elastic Upgrade Assistant. Elasticsearch requests made by the Monitoring or Central Management features that are not directly user-configurable will include a special product origin header so that Upgrade Assistant can avoid calling out deprecations that are not user-actionable. https://github.com/elastic/logstash/pull/13563[#13563]
```